pkgs/applications/blockchains/trezor-suite/default.nix

```diff
@@ -8,7 +8,7 @@
 
 let
   pname = "trezor-suite";
-  version = "23.4.2";
+  version = "23.10.1";
   name = "${pname}-${version}";
```

Comment:
All the appimage tools now support pname+version and we no longer need to construct name manually.